src/SignalR/clients/csharp/Http.Connections.Client/ref/Microsoft.AspNetCore.Http.Connections.Client.csproj

```diff
@@ -1,18 +0,0 @@
-<!-- This file is automatically generated. -->
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Hrm, I wonder why we had this. It wasn't in the shared framework before... (I just checked)